exercises/linked-list/example.py

```diff
@@ -45,3 +45,21 @@ def unshift(self, value):
             new_node.next = self.head
             self.head.prev = new_node
             self.head = new_node
+
+    def __len__(self):
```

Comment:
Thanks for the review. I didn't spot your pull request until after I'd submitted mine - I'm still getting used to github.

My approach is more 'self contained' but worse from a big O perspective. From a teaching perspective it would be good to include both and document the pros/cons of each approach.